### PR TITLE
Add a profile of  neil AIvares to the community member

### LINKED ISSUE #1359: [Community] Member Profile: Neil Alvares
**Current Behaviour**

@Neil-140301 has been a consistent contributor and community member.

**Desired Situation**
Let's recognizing @Neil-140301 as a contributor and community member by creating a profile on https://layer5ng.netlify.app/community/members.

- GitHub: Neil-140301 
- Twitter: 
- LinkedIn: neil-alvares-2986561a0
- Link to profile picture:

---

**Contributor Resources**

The layer5.io website, which uses GatsbyJS and GitHub Pages. Site content is found under the <a href="https://github.com/layer5io/layer5/tree/layer5-ng">`layer5-ng` branch</a>. See <a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md">`layer5` contributing instructions</a>.

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
Added Neil Ivare to the community profile
This PR fixes #1359

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thank you, @Anita-ihuman. You'll probably want to rename the two files to be "neil-alvares" instead of "neil-ivare".

**Review by leecalcote — APPROVED** [review]

**Review by Neil-140301 — COMMENTED** [review]
just a small comment, I can't see the links to my LinkedIn and GitHub profiles in the preview.
Thank you @Anita-ihuman 😁

File: src/collections/members/neil-alvares/index.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: person
```
Comment (Tanuj22) [inline src/collections/members/neil-alvares/index.mdx:2]: @Anita-ihuman This is the previous template. Follow the template for the `layer5-ng` site.

**Comment by leecalcote** [comment]
@Anita-ihuman here is a link to the Layer5-NG community member profile template - https://github.com/layer5io/layer5/blob/layer5-ng/src/collections/members/_member-profile-template/index.mdx

File: src/collections/members/neil-alvares/index.mdx (hunk 1 above)
Comment (leecalcote) [reply src/collections/members/neil-alvares/index.mdx:2]: template here - https://github.com/layer5io/layer5/blob/layer5-ng/src/collections/members/_member-profile-template/index.mdx

**Comment by Tanuj22** [comment]
@Anita-ihuman can you update this according to the tmeplate

**Comment by Anita-ihuman** [comment]
@Neil-140301  please can you give me your information on these, please.?
location: 
bio:  
twitter: 
badges:

**Comment by Neil-140301** [comment]
location: Pune, India
Bio: 'I am a sophomore at the Indian Institute of Technology Jodhpur studying Electrical Engineering. I am passionate about full-stack web development and UI/UX design. I don't play sports and am a Disney and Marvel fan. I look forward to lots of learning and building with my time here.'
Twitter: I don't have one. here's my [linkedin](https://www.linkedin.com/in/neil-alvares/)
badges : none

**Review by Anita-ihuman — COMMENTED** [review]
The update has been made according to the profile template.

**Review by Tanuj22 — APPROVED** [review]

**Comment by leecalcote** [comment]
Thank you, @Anita-ihuman. 😄

**Comment by leecalcote** [comment]
Congratulations, @Neil-140301!

**Pull request merged**